oshi-core/src/test/java/oshi/hardware/CentralProcessorTest.java

```diff
@@ -49,45 +49,60 @@ public void testCentralProcessor() {
         CentralProcessor p = si.getHardware().getProcessor();
 
         ProcessorIdentifier pi = p.getProcessorIdentifier();
-        assertNotNull(pi.getVendor());
-        assertTrue(pi.getVendorFreq() == -1 || pi.getVendorFreq() > 0);
-        assertNotNull(pi.getName());
-        assertNotNull(pi.getIdentifier());
-        assertNotNull(pi.getProcessorID());
-        assertNotNull(pi.getStepping());
-        assertNotNull(pi.getModel());
-        assertNotNull(pi.getFamily());
-        assertFalse(Util.isBlank(pi.getMicroarchitecture()));
+        assertNotNull("Processor Identifier's vendor shouldn't be Null", pi.getVendor());
+        assertTrue("Processor Identifier's vendor frequency should either be -1, or a greater then 0",
```

Comment:
Minor point: should be "than" not "then"

Reply:
Will fix.